scripts/dovecot/README.md

```diff
@@ -0,0 +1,83 @@
+## Introduction to custom Dovecot builds
+
+Chatmail servers use a custom Debian build of the IMAP 'dovecot' server software because 
+
+a) Dovecot developers did not yet merge a [pull request](https://github.com/dovecot/core/pull/216)
+   which majorly speeds up message delivery by removing a hardcoded 0.5 second delay 
+   on relaying incoming messages.  
+
+b) Even if merged, it would take years for it to reach Debian stable,
+   the distribution that chatmail deployment uses. 
```

Comment:
We don't necessarily use Debian stable, we just depend on apt.

Reply:
well, that is bad then. because our package is built for debian-stable

Reply:
do we really want to open the rabbit-hole of supporting multiple distros? at that point we should've written a docker-compose script instead

Reply:
I'm fine with removing the last sentence, but if we want to support other Distros than Debian 12, we should build packages for them. See https://wiki.debian.org/DontBreakDebian